etcd/etcd-practices.adoc

```diff
@@ -4,4 +4,14 @@ include::_attributes/common-attributes.adoc[]
 = Recommended etcd practices
 :context: etcd-practices
 
-// This assembly will contain modules to provide recommended etcd practices.
\ No newline at end of file
+toc::[]
+
+To ensure optimal performance and scalability for etcd in {product-title}, you can complete the following practices.
```

Comment:
This has the exact same intro sentence as the etcd-performance.adoc assembly. This should be something different, so people should know what to expect from this one vs. the other one.